Documentation/Books/AQL/Operations/Insert.md

```diff
@@ -24,7 +24,7 @@ traversal operations, or AQL functions that can read documents.
 The syntax for an insert operation is:
 
 ```
-INSERT document IN collection options
+INSERT document IN collection [ OPTIONS {options} ]
```

Comment:
please use a headline like here: https://docs.arangodb.com/devel/AQL/Operations/Replace.html#setting-query-options to explain what options you can set.

Reply:
There is already such a headline. How about the following changes though?

````diff
diff --git a/Documentation/Books/AQL/Operations/Insert.md b/Documentation/Books/AQL/Operations/Insert.md
index 1932110..8d1e18d 100644
--- a/Documentation/Books/AQL/Operations/Insert.md
+++ b/Documentation/Books/AQL/Operations/Insert.md
@@ -24,10 +24,10 @@ traversal operations, or AQL functions that can read documents.
 The syntax for an insert operation is:
 
 ```
-INSERT document IN collection [ OPTIONS {options} ]
+INSERT document INTO collection [ OPTIONS options ]
 ```
 
-**Note**: The *INTO* keyword is also allowed in the place of *IN*.
+**Note**: The *IN* keyword is allowed in place of *INTO* and has the same meaning.
 
 *collection* must contain the name of the collection into which the documents should
 be inserted. *document* is the document to be inserted, and it may or may not contain
@@ -37,7 +37,7 @@ revision number for the document.
 
 ```js
 FOR i IN 1..100
-  INSERT { value: i } IN numbers
+  INSERT { value: i } INTO numbers
 ```
 
 When inserting into an [edge collection](../../Manual/Appendix/Glossary.html#edge-collection),
@@ -47,13 +47,16 @@ it is mandatory to specify the attributes *_from* and *_to* in document:
 FOR u IN users
   FOR p IN products
     FILTER u._key == p.recommendedBy
-    INSERT { _from: u._id, _to: p._id } IN recommendations
+    INSERT { _from: u._id, _to: p._id } INTO recommendations
 ```
 
 Setting query options
 ---------------------
 
-*options* can be used to suppress query errors that may occur when violating unique
+The *OPTIONS* keyword followed by an object with query options can optionally
+be provided in an *INSERT* operation.
+
+It can be used to suppress query errors that may occur when violating unique
 key constraints:
 
 ```js
@@ -103,7 +106,7 @@ the database (e.g. `_id`, `_key`, `_rev`).
 
 
 ```js
-INSERT document IN collection RETURN NEW
+INSERT document INTO collection RETURN NEW
 ```
 
 Following is an example using a variable named `inserted` to return the inserted
@@ -112,7 +115,7 @@ documents. For each inserted document, the document key is returned:
 ```js
 FOR i IN 1..100
   INSERT { value: i }
-  IN users 
-  LET inserted = NEW 
+  INTO users
+  LET inserted = NEW
   RETURN inserted._key
 ```
```

I removed the `{}` around *options* because for *document* we don't write them out either.
INTO should be used as default, because it is better English.